src/seedsigner/models/psbt_parser.py

```diff
@@ -182,6 +182,10 @@ def _parse_outputs(self):
                 })
                 self.change_amount += self.psbt.tx.vout[i].value
 
+            elif self.psbt.tx.vout[i].value == 0 and self.psbt.tx.vout[i].script_pubkey.script_type() is None and self.psbt.tx.vout[i].script_pubkey.data:
+                # OP_RETURN
+                self.op_return = self.psbt.tx.vout[i].script_pubkey.data[3:].decode('UTF-8')
```

Comment:
wondering what these first 3 bytes are?
* op_return?
* compact size?
* ???
Can we assert some of them... or use them just above to know for sure it is op_return?

Reply:
Ah, yes, I was being so dumb!!

First 3 bytes = OP_RETURN + OP_PUSHDATA1 + len(payload)

Updated in latest commit.